app/views/needs/show.html.erb

```diff
@@ -96,6 +96,20 @@
   </h2>
 
   <%= form_with(model: [@need, @need.notes.build], local: true) do |form| %>
+    <div class="field">
+      <% nnc = needs_note_categories.find {|e| e[1][:stored_as] == @need.category}  %>
```

Comment:
Let's have a look at the figma designs and see how they handle rendering this so @jhackett1 doesn't have to fix it up if we can handle it. Also note that choosing a category should be optional (hopefully that's reflected in the designs), so you can log a note that isn't about a phone call.